_projects/interf_aware_sched.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page_project
+title: Towards Interference-aware scheduling in HPC systems
+date: 2016-03-21
+updated:
+navbar: Research
+subnavbar: Projects
+project_url:
+status: running
+topics: storage
+  -
```

Comment:
Unfortunately, this is causing an error. Either delete the line with the single dash or put the project ID `storage` after the dash:
```
[...]
status: running
topics: storage
keywords:
[...]
```
or
```
[...]
status: running
topics:
  - storage
keywords:
[...]
```